feature-utils/poly-look/test/react-components/filterChips.test.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import FilterChips from "../../src/react-components/filterChips/filterChips.jsx";
+
+const chipsContent = [...Array(3).keys()].map((i) => `chip${i}`);
+const mockedHandleClick = jest.fn();
+const allChipContent = "All";
+// const mockedActiveChips = [chipsContent[1], chipsContent[2]];
+
+const component = (
+  <FilterChips
+    chipsContent={chipsContent}
+    onChipClick={mockedHandleClick}
+    allChip={{ translation: allChipContent }}
+  />
+);
+
+it("Chips present in document", () => {
```

Comment:
We normally group these tests in a "describe" sentence, but as you have seen, it's not really necessary.